src/compiler/build.jl

```diff
@@ -1,8 +1,28 @@
-using MutableArithmetics
-const MA = MutableArithmetics
+function toqubo_build!(model::VirtualModel{T}, arch::AbstractArchitecture) where {T}
+    #  Assemble Objective Function 
+    toqubo_hamiltonian!(model, arch)
+
+    #  Quadratization Step 
+    toqubo_quadratize!(model, arch)
 
-function toqubo_hamiltonian!(model::VirtualQUBOModel{T}, ::AbstractArchitecture) where {T}
-    copy!(model.H, model.f)
+    #  Write to MathOptInterface 
+    toqubo_output!(model, arch)
+
+    return nothing
+end
+
+function toqubo_hamiltonian!(model::VirtualModel{T}, ::AbstractArchitecture) where {T}
+    empty!(model.H)
+
+    # TODO: Make this 'sizehint!' more precise
+    # IDEA:
```

Comment:
I don't understand this TODO, as it seems to be solved in the line below. If it isn't, add it to the issues that were included elsewhere and we'll work on it on a future PR.

Reply:
Adjusted